R/mod_indvExpPatEvents.R

```diff
@@ -108,8 +108,8 @@ mod_indvExpPatEvents_server <- function(input, output, session,
           date_cols <- c("Start of Event","End of Event")
         }
         else{
-          date_cols <- "Date of Event"
-          tab <- uni_rec %>% select(EVENTTYP, START, DECODE)
+          date_cols <- c("Date of Event", "End Date of Event")
```

Comment:
Since we are displaying "End Date of Event", can we change "Date of Event" to "Start Date of Event"?

Reply:
I'd consider something that rolls off the tongue better, like "Event Start Date" and "Event End Date".

Reply:
Since the table is clearly labelled an events table, I am going to go with the even simpler "Start Date" and "End Date" unless told otherwise.